torchao/quantization/quant_api.py

```diff
@@ -234,6 +234,7 @@ def _is_linear(mod, *args):
         and not isinstance(mod.weight, AffineQuantizedTensor)
         and not isinstance(mod.weight, LinearActivationQuantizedTensor)
         and not isinstance(mod.weight, AffineFakeQuantizedTensor)
+        and not isinstance(mod, nn.modules.linear.NonDynamicallyQuantizableLinear)
```

Comment:
nit: maybe add a Note here, to say that if MHA is refactored to not use this module we'd need a different solution